source/solvers/postprocessing_cfd.cc

```diff
@@ -434,14 +435,30 @@ calculate_forces(
 {
   const FESystem<dim, dim> fe = dof_handler.get_fe();
 
-  double viscosity = physical_properties.fluids[0].viscosity;
+  double viscosity;
+  if (!physical_properties.non_newtonian_flow)
+    viscosity = physical_properties.fluids[0].viscosity;
+
+  // Non newtonian properties
+  std::unique_ptr<RheologicalModel<dim>> rheological_model;
+  if (physical_properties.non_newtonian_flow)
+    {
+      // if (physical_properties.non_newtonian_parameters.model ==
+      // Parameters::NonNewtonian::Model::Carreau)
+      //{
+      rheological_model = std::make_unique<Carreau<dim>>(
+        physical_properties.non_newtonian_parameters);
+      //}
```

Comment:
Those these comments are still necessary?

Reply:
I think we should keep them as long as there is not more than 1 rheological model